tests/images.sh

```diff
@@ -3,8 +3,18 @@
 set -euo pipefail
 
 export PATH
-if [[ -d "$(realpath "$(dirname "$0")/../target")" ]]; then
-    PATH="$(realpath "$(dirname "$0")/../target/${PROFILE:-debug}"):$PATH"
+bindir="$(realpath -m "$(dirname "$0")/../target/${PROFILE:-debug}")"
+if [[ -x "$bindir/coreos-installer" ]]; then
+    PATH="$bindir:$PATH"
+else
```

Comment:
Should this be e.g.

```
elif command -v coreos-installer >/dev/null; then
```

?

Reply:
I was relying on `which` to error out if the command doesn't exist, but your approach is clearer.